cob_description/urdf/cob4_base/base.urdf.xacro

```diff
@@ -19,6 +19,9 @@
 
   <xacro:macro name="base" params="name use_old_joint_names">
 
+    <xacro:property name="drive_vel" value="19.95" />
+    <xacro:property name="steer_vel" value="117.81" />
```

Comment:
could you please add the units (and maybe the relevant canopen objects) as a comment